Tests/PhpUnit/AbstractPrependExtensionTestCaseTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Matthias\DependencyInjectionTests\Test\DependencyInjection;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Matthias\SymfonyDependencyInjectionTest\Tests\Fixtures\MatthiasDependencyInjectionTestExtension;
+use PHPUnit\Framework\ExpectationFailedException;
+
+class AbstractPrependExtensionTestCaseTest extends AbstractExtensionTestCase
+{
+    protected function getContainerExtensions()
```

Comment:
Ha, good point ;) I've added these kind of `protected` methods at the top to show that they are being explicitly extended/implemented. It's a habit, I wasn't very much aware of could be considered conflicting. Not a really big deal anyway :)